libs/filamat/src/ShaderMinifier.cpp

```diff
@@ -137,7 +137,7 @@ namespace {
  * - Remove leading white spaces at the beginning of each line
  * - Remove empty lines
  */
-std::string ShaderMinifier::removeWhitespace(const std::string& s) const {
+std::string ShaderMinifier::removeWhitespace(const std::string& s, bool mergeBraces) const {
```

Comment:
why not doing this regardless of the optimization mode?

Reply:
Because it actually makes things worse for the other cases.